src/server/pegasus_mutation_duplicator.h

```diff
@@ -97,5 +98,8 @@ class pegasus_mutation_duplicator : public dsn::replication::mutation_duplicator
 // calculates the hash value from the write's hash key.
 extern uint64_t get_hash_from_request(dsn::task_code rpc_code, const dsn::blob &request_data);
 
+const std::set<int> ingnored_rpc_code = {dsn::apps::RPC_RRDB_RRDB_DUPLICATE,
```

Comment:
Nobody else use it, you can move it to the cpp file, define it as a local static variable in pegasus_mutation_duplicator::duplicate().